src/components/SignIn/SignInPage.js

```diff
@@ -1,25 +1,45 @@
 import './SignInPage.css'
 import {Link} from 'react-router-dom'
+import { Component } from 'react';
 
-function SignInPage() {
-  return (
-    <div className="SignInPage">
-      <form>
-        <label>
-          email
-          <input type='email'/>
-        </label>
-        <label>
-          password
-          <input type='password'/>
-        </label>
-        <button> Submit </button>
-      </form>
-      <footer className="SignInPage-footer">
-        <Link to='/'><button> Take Me Back </button></Link>
-      </footer>
-    </div>
-  );
+class SignInPage extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      email: '',
```

Comment:
Just a question for my own learning. Is this where you could put a default value for the field?

Reply:
This is actually temporary data storage that is currently being mutated by a function when the user fills out their form.
So this isn;t data a user actually sees, it is logic handling for the ux elements to interact with.